### PR TITLE
Divide opset1 tests

### PR DESCRIPTION
**Pull request by AnastasiaKazantaeva**
Added ability to filter only opset1 backend tests.  Wrote pytest test for its execution.

### CODE REVIEW

File: test/util/backend_utils.hpp [hunk 1]
```diff
@@ -20,187 +20,210 @@
...
+    {
+    private:
+        InferenceEngine::CNNNetwork network;
+        std::string device;
+
+    public:
+        Executable(std::shared_ptr<Function> func, std::string _device)
         {
+            auto opset = ngraph::get_opset1();
+            bool all_opset1(true);
```
Comment (gcwenger) [inline test/util/backend_utils.hpp]:
```suggestion
            bool all_opset1 = true;
```

File: test/util/backend_utils.hpp [hunk 2]
```diff
@@ -20,187 +20,210 @@
...
+            for (auto& node : func->get_ops())
+            {
+                if (!opset.contains_op_type(node.get()))
+                    all_opset1 = false;
```
Comment (gcwenger) [inline test/util/backend_utils.hpp]:
```suggestion
                    {
                        all_opset1 = false;
                        break;
                    }
```

File: test/util/backend_utils.hpp [hunk 3]
```diff
@@ -20,187 +20,210 @@
...
+            }
+
+            if (!all_opset1)
+            {
+                std::cout << "UNSUPPORTED OPS DETECTED!" << std::endl;
+                THROW_IE_EXCEPTION << "Exit from test";
+            }
+            else
```
Comment (gcwenger) [inline test/util/backend_utils.hpp]: Remove `else`. Code flow would not continue past `THROW_IE_EXCEPTION` so no need for `else`.

File: test/util/backend_utils.hpp [hunk 4]
```diff
@@ -20,187 +20,210 @@
...
+            void write(const void* p, size_t n) override
+            {
+                const int8_t* v = (const int8_t*)p;
+                if (v == nullptr)
+                    return;
+                data.resize(n);
+                for (size_t i = 0; i < n; ++i)
+                {
+                    data[i] = v[i];
+                }
```
Comment (gcwenger) [inline test/util/backend_utils.hpp]: std::copy(v, v+n, data.begin());

File: test/util/backend_utils.hpp [hunk 5]
```diff
@@ -20,187 +20,210 @@
...
+            }
+            /// \brief Read bytes directly from the tensor
+            /// \param p Pointer to destination for data
+            /// \param n Number of bytes to read, must be integral number of elements.
+            void read(void* p, size_t n) const override
+            {
+                int8_t* v = (int8_t*)p;
+                if (v == nullptr)
+                    return;
+                for (size_t i = 0; i < n; ++i)
```
Comment (gcwenger) [inline test/util/backend_utils.hpp]: First assert that `n == data.size()` then:
`std::copy(data.begin(), data.end(), v);`

File: test/util/backend_utils.hpp [hunk 6]
```diff
@@ -20,187 +20,210 @@
...
-
-        float* output_ptr = output->buffer().as<float*>();
-        // TODO: how to get size without explicit calculation?
-        size_t size = 1;
-        for (const auto& dim : output->getTensorDesc().getDims())
+        std::shared_ptr<ov_runtime::Tensor> create_tensor(ngraph::element::Type type,
+                                                          ngraph::Shape shape)
         {
-            size *= dim;
+            return std::shared_ptr<ov_runtime::Tensor>(new ov_runtime::Tensor(type, shape));
```
Comment (gcwenger) [inline test/util/backend_utils.hpp]:
```suggestion
            return std::make_shared<ov_runtime::Tensor>(type, shape);
```

File: test/util/backend_utils.hpp [hunk 7]
```diff
@@ -20,187 +20,210 @@
...
-    {
-        auto tensor = std::shared_ptr<ov_runtime::Tensor>(new ov_runtime::Tensor(type, shape));
-        size_t size = 1;
-        for (auto x : shape)
+        template <typename T>
+        std::shared_ptr<ov_runtime::Tensor>
+            create_tensor(ngraph::element::Type type, ngraph::Shape shape, T* data)
         {
-            size *= x;
+            auto tensor = std::shared_ptr<ov_runtime::Tensor>(new ov_runtime::Tensor(type, shape));
```
Comment (gcwenger) [inline test/util/backend_utils.hpp]:
```suggestion
            auto tensor = std::make_shared<ov_runtime::Tensor>(type, shape);
```

File: test/util/backend_utils.hpp [hunk 8]
```diff
@@ -20,187 +20,210 @@
...
-    bool supports_dynamic_tensors() { return true; }
-    std::shared_ptr<Executable> compile(std::shared_ptr<Function> func)
-    {
-        return std::shared_ptr<Executable>(new Executable(func, device));
-    }
-};
+        template <class T>
+        std::shared_ptr<ov_runtime::Tensor> create_tensor(ngraph::Shape shape)
+        {
+            return std::shared_ptr<ov_runtime::Tensor>(
```
Comment (gcwenger) [inline test/util/backend_utils.hpp]:
```suggestion
            return std::make_shared<ov_runtime::Tensor>(
     	                ngraph::element::from<T>(), shape);
```

File: test/util/backend_utils.hpp [hunk 9]
```diff
@@ -20,187 +20,210 @@
...
+                new ov_runtime::Tensor(ngraph::element::from<T>(), shape));
```
Comment (gcwenger) [inline test/util/backend_utils.hpp]:
```suggestion
```

File: test/util/backend_utils.hpp [hunk 10]
```diff
@@ -20,187 +20,210 @@
...
-    for (size_t i = 0; i < v.size(); ++i)
-    {
-        v[i] = tv->data[i];
-    }
-    return v;
-}
+        std::shared_ptr<ov_runtime::Tensor> create_dynamic_tensor(ngraph::element::Type type,
+                                                                  ngraph::PartialShape shape)
+        {
+            return std::shared_ptr<ov_runtime::Tensor>(new ov_runtime::Tensor(type, shape));
```
Comment (gcwenger) [inline test/util/backend_utils.hpp]:
```suggestion
            return std::make_shared<ov_runtime::Tensor>(type, shape);
```

File: test/util/backend_utils.hpp [hunk 11]
```diff
@@ -20,187 +20,210 @@
...
-{
-    t->data.resize(data.size());
-    for (size_t i = 0; i < data.size(); ++i)
-    {
-        t->data[i] = data[i];
-    }
+        bool supports_dynamic_tensors() { return true; }
+        std::shared_ptr<Executable> compile(std::shared_ptr<Function> func)
+        {
+            return std::shared_ptr<Executable>(new Executable(func, device));
```
Comment (gcwenger) [inline test/util/backend_utils.hpp]:
```suggestion
            return std::make_shared<Executable>(func, device);
```

**Comment by ilyachur** [comment]
@gcwenger Please take a look one more time.

**Pull request merged**